cmd/openfga/openfga.go

```diff
@@ -155,8 +162,21 @@ func buildService(logger logger.Logger) (*service, error) {
 		return nil, fmt.Errorf("storage engine '%s' is unsupported", config.DatastoreEngine)
 	}
 
-	var authenticator authentication.Authenticator
+	var tlsConfig *server.TLSConfig
+	if config.EnableTLS {
+		if config.TLSCertPath == "" || config.TLSKeyPath == "" {
+			return nil, errFailedToSetTLSVariables
+		}
+		tlsConfig = &server.TLSConfig{
+			CertPath: config.TLSCertPath,
+			KeyPath:  config.TLSKeyPath,
+		}
+		logger.Info("will serve TLS")
```

Comment:
```suggestion
		logger.Info("TLS enabled, serving connections using the provided certificate...")
```

Reply:
I'll use this but without the ellipses 🙂